core/crypto/value_test.go

```diff
@@ -0,0 +1,46 @@
+package crypto
+
+import "testing"
+
+func TestGenerateValue(t *testing.T) {
+	type args struct {
+		template string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "Success case for alphanumeric",
+			args: args{
+				template: `foo[\w]{8}bar`,
+			},
+			wantErr: false,
```

Comment:
Can we also check return value?

Reply:
No, because this output is randomly generated